src/ResourceManager/Automation/Commands.Automation/packages.config

```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.Management.Automation" version="3.4.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.ResourceManager" version="2.0.0-preview" targetFramework="net452" />
```

Comment:
Keep in mind you won't be able to use this when merging back to our main branch.